discretisedfield/plotting/util.py

```diff
@@ -112,3 +113,33 @@ def hls2rgb(hue, lightness=None, saturation=None, lightness_clim=None):
     )
 
     return rgb.squeeze()
+
+
+def arrow():
+    return pv.Arrow(
+        tip_radius=0.18,
+        tip_length=0.4,
+        tip_resolution=10,
+        shaft_resolution=10,
+        shaft_radius=0.05,
+        start=(-0.5, 0, 0),
+    )
+
+
+def cone():
+    return pv.Cone(
+        center=(-0.5, 0, 0),
+    )
+
+
+def _pyvista_save_to_file(filename, plotter):
+    extension = filename.split(".")[-1] if "." in filename else None
+    if extension in ["png", "jpeg", "jpg", "bmp", "tif", "tiff"]:
+        plotter.screenshot(filename=filename)
+    elif extension in ["svg", "eps", "ps", "pdf", "tex"]:
+        plotter.save_graphic(filename=filename)
+    else:
+        raise ValueError(
+            f"{extension} extension is not supported. The supported formats are"
+            " png, jpeg, jpg, bmp, tif, tiff, svg, eps, ps, pdf, and txt."
+        )
```

Comment:
I suggest we use variables for "screenshot" and "graphic" formats and re-use these lists for the error message to ensure the two match (e.g. `txt` seems to be a typo and should be `tex`?). This will also make future updates easier and less error-prone.